src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts

```diff
@@ -162,7 +242,9 @@ export var hideActionDirective = (
 ) => {
     return {
         restrict: "E",
-        template: "<a class=\"{{class}}\" href=\"\" data-ng-click=\"hide();\">{{ 'TR__HIDE' | translate }}</a>",
+        transclude: true,
+        template: "<a class=\"{{class}}\" href=\"\" data-ng-click=\"hide();\"><ng-transclude></ng-transclude>" +
+            "{{ 'TR__HIDE' | translate }}</a>",
```

Comment:
Good question:
1. `modalActionDirective` and `viewActionDirective` use different icons for different actions. `ng-transclude` comes in handy to get those different icons into them.
2. I strove to make the different action directives as similar as possible.
3. En passant, `ng-transclude` includes a little whitespace between icon and text.

You're welcome to relinquish `ng-transclude` where possible (you compromise (2) and need to solve (3) then, though).